2020-04-25 20:08:57,234 - experiment_impact_tracker.compute_tracker.ImpactTracker - ERROR - Encountered exception within power monitor thread!
2020-04-25 20:08:57,236 - experiment_impact_tracker.compute_tracker.ImpactTracker - ERROR -   File "/home/sai/miniconda3/envs/bertexp/lib/python3.7/site-packages/experiment_impact_tracker/compute_tracker.py", line 105, in launch_power_monitor
    _sample_and_log_power(log_dir, initial_info, logger=logger)
  File "/home/sai/miniconda3/envs/bertexp/lib/python3.7/site-packages/experiment_impact_tracker/compute_tracker.py", line 55, in _sample_and_log_power
    required_headers = _get_compatible_data_headers(get_current_region_info_cached()[0])
  File "/home/sai/miniconda3/envs/bertexp/lib/python3.7/site-packages/experiment_impact_tracker/compute_tracker.py", line 120, in _get_compatible_data_headers
    if not compatability_fn(region=region):
  File "/home/sai/miniconda3/envs/bertexp/lib/python3.7/site-packages/experiment_impact_tracker/gpu/nvidia.py", line 31, in is_nvidia_compatible
    stdout, stderror = p.communicate()
  File "/home/sai/miniconda3/envs/bertexp/lib/python3.7/subprocess.py", line 951, in communicate
    stdout = self.stdout.read()

2020-04-25 21:00:53,339 - experiment_impact_tracker.compute_tracker.ImpactTracker - ERROR - Encountered exception within power monitor thread!
2020-04-25 21:00:53,347 - experiment_impact_tracker.compute_tracker.ImpactTracker - ERROR -   File "/home/sai/miniconda3/envs/bertexp/lib/python3.7/site-packages/experiment_impact_tracker/compute_tracker.py", line 105, in launch_power_monitor
    _sample_and_log_power(log_dir, initial_info, logger=logger)
  File "/home/sai/miniconda3/envs/bertexp/lib/python3.7/site-packages/experiment_impact_tracker/compute_tracker.py", line 69, in _sample_and_log_power
    results = header["routing"]["function"](process_ids, logger=logger, region=initial_info['region']['id'], log_dir=log_dir)
  File "/home/sai/miniconda3/envs/bertexp/lib/python3.7/site-packages/experiment_impact_tracker/cpu/common.py", line 28, in get_cpu_freq
    return [x._asdict() for x in psutil.cpu_freq(percpu=True)]
  File "/home/sai/miniconda3/envs/bertexp/lib/python3.7/site-packages/psutil/__init__.py", line 1877, in cpu_freq
    ret = _psplatform.cpu_freq()
  File "/home/sai/miniconda3/envs/bertexp/lib/python3.7/site-packages/psutil/_pslinux.py", line 696, in cpu_freq
    curr = cat(pjoin(path, "scaling_cur_freq"), fallback=None)
  File "/home/sai/miniconda3/envs/bertexp/lib/python3.7/site-packages/psutil/_pslinux.py", line 300, in cat
    return f.read().strip()

